Mathlib/CategoryTheory/Adjunction/Mates.lean

```diff
@@ -108,13 +111,58 @@ def transferNatTrans : (G ⋙ L₂ ⟶ L₁ ⋙ H) ≃ (R₁ ⋙ G ⟶ H ⋙ R
       h.naturality, -Functor.map_comp, ← Functor.map_comp_assoc G, R₂.map_comp]
 #align category_theory.transfer_nat_trans CategoryTheory.transferNatTrans
 
+-- ER: A new construction with a new name.
+def Mates :
```

Comment:
If you write `@[simps] def Mates :` here, then lean will automatically generate some lemmas describing the projections for you. (In fact, `transferNatTrans` should have been tagged simps too, but I don't think it existed when I wrote `transferNatTrans`!)

Reply:
Thanks I'll do this.